addons/web_editor/models/ir_qweb.py

```diff
@@ -389,7 +389,7 @@ def from_html(self, model, field, element):
             content.append(element.text)
         content.extend(html.tostring(child, encoding='unicode')
                        for child in element.iterchildren(tag=etree.Element))
-        return '\n'.join(content)
+        return ''.join(content)
```

Comment:
Won't the suggested change lead to the following?
Input:
```html
<div class="container">
    <h1>My Title</h1>
    <p>Such an interesting content.</p>
</div>
```
Output after the change:
```html
My TitleSuch an interesting content.
```
Whereas before it used to be:
```html
My Title
Such an interesting content.
```